src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -534,27 +543,27 @@ public void addOnRankedMatchNotificationListener(Consumer<MatchmakerInfoMessage>
               gameLaunchMessage.getArgs().add("/players " + gameLaunchMessage.getExpectedPlayers());
               gameLaunchMessage.getArgs().add("/startspot " + gameLaunchMessage.getMapPosition());
 
-              startGame(gameLaunchMessage, faction, RatingMode.LADDER_1V1);
+              startGame(gameLaunchMessage, gameLaunchMessage.getFaction(), NONE); // TODO: rating mode
```

Comment:
This one seems needed as well unless I am missing something

Reply:
This is for displaying the right rating in the game. We should grab the appropriate leaderboard from the queue information once #1791 is implemented.